hasjob/views/listing.py

```diff
@@ -953,6 +925,19 @@ def newjob():
         if g.user:
             # form.poster_name.data = g.user.fullname  # Deprecated 2013-11-20
             form.poster_email.data = g.user.email
+
+    # Job Reposting
+    if request.method == 'GET' and request.args.get('template'):
+        archived_post = JobPost.get(request.args.get('template'))
+        if not archived_post:
+            abort(404)
+        if not archived_post.admin_is(g.user):
+            abort(403)
+        if not archived_post.is_old():
+            flash("This listing is currently active and cannot be posted again.")
```

Comment:
We changed terminology from `listing` to `post` a while ago. This should be `post` too (old habit dying hard).